app/script/components/groupVideoGrid.js

```diff
@@ -57,35 +58,26 @@ z.components.GroupVideoGrid = class GroupVideoGrid {
 
   scaleVideos(rootElement) {
     const elements = Array.from(rootElement.querySelectorAll('.group-video-grid__element'));
-    const setScale = (video, containerRatio) => {
-      const fullHeightClass = 'group-video-grid__element-video--fill-height';
-      const fullWidthClass = 'group-video-grid__element-video--fill-width';
-      const videoRatio = video.videoWidth / video.videoHeight;
-      video.classList.remove(fullHeightClass);
-      video.classList.remove(fullWidthClass);
-
-      const isPortrait = videoRatio > containerRatio || video.videoWidth < video.videoHeight;
-      const fillClasses = isPortrait ? fullHeightClass : fullWidthClass;
-      video.classList.add(fillClasses);
+    const setScale = video => {
+      const isPortrait = video.videoHeight > video.videoWidth;
+      video.classList.toggle(GroupVideoGrid.CONFIG.CONTAIN_CLASS, isPortrait);
     };
 
     elements.forEach(element => {
-      const containerRect = element.getBoundingClientRect();
-      const containerRatio = containerRect.width / containerRect.height;
-      const videoElement = element.querySelector('video');
-      const handleLoadEvent = event => {
-        const video = event.target;
-        setScale(video, containerRatio);
-      };
-
-      if (videoElement.videoWidth > 0) {
-        setScale(videoElement, containerRatio);
+      const video = element.querySelector('video');
```

Comment:
Could we stick to the `videoElement` semantic, I find `video` to be a bit to generic